app/collins/controllers/IpmiApi.scala

```diff
@@ -85,4 +88,8 @@ trait IpmiApi {
     )
   }(Permissions.IpmiApi.UpdateIpmi)
 
+  // GET /api/ipmi/pools
+  def getIpmiAddressPools(all: String) =
+    GetPoolsAction(Truthy(all), Permissions.IpAddressApi.UpdateAddress, this)
```

Comment:
this permission seems incorrect. We should use a separate "ipmi.get address pools" perm, and perhaps a separate one for ipmi network view from normal address pool view?